hack/dockerfiles/lint.Dockerfile

```diff
@@ -5,6 +5,9 @@ ARG ALPINE_VERSION=3.19
 ARG XX_VERSION=1.4.0
 ARG PROTOLINT_VERSION=0.45.0
 ARG GOLANGCI_LINT_VERSION=1.57.1
+ARG GOPLS_VERSION=v0.20.0
+# disabled: deprecated unusedvariable simplifyrange
```

Comment:
simplifyrange can't be enabled as secrets.pb.go generated code invalidates it atm. and there doesn't seem to be a way to disable checks on these files like you can do in golangci-lint.

unusedvariable disabled because it is disabled in gopls by default as well. It doesn't actually match anything in buildkit.

deprecated should be manually checked periodically but I don't think we can keep it as a requirement in CI.

Reply:
> deprecated should be manually checked periodically but I don't think we can keep it as a requirement in CI.

If you want this to be included on schedule on ci this is possible.